pulp_service/pulp_service/app/models.py

```diff
@@ -174,3 +175,4 @@ class VulnerabilityReport(models.Model):
 
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     vulns = models.JSONField()
+    pulp_domain = models.ForeignKey("core.Domain", default=get_domain_pk, on_delete=models.PROTECT)
```

Comment:
Not sure if you want this to be `PROTECT` for `on_delete`. It would require you to have to delete all VulnerabilityReports before deleting a domain. Maybe this is the behavior you want, but I think having `CASCADE` would be simpler (there are already protections on other user objects)